common/scripts/default-mesh.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -eu
+
+# Check if proper default mesh class is set
+if [[ "$MESH_CLASS" != "gs" && "$MESH_CLASS" != "edge" ]]; then
+  echo "Wrong mesh class, check container env parameters"
+  exit 1
+else
+  echo "Mesh class: " $MESH_CLASS
```

Comment:
Lot of double quotes missing around the variables (to avoid word splitting).  Please check the file with shellcheck tool.

Reply:
Done according to the shellcheck recommendations.